sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -165,19 +165,25 @@ case class QualifiedColType(
  *                             would have Map('a' -> Some('1'), 'b' -> None).
  * @param ifPartitionNotExists If true, only write if the partition does not exist.
  *                             Only valid for static partitions.
+ * @param byName               If true, reorder the data columns to match the column names of the
+ *                             target table. The field name of the datasource to ensure that the
```

Comment:
I think we can remove `The field name ...`

Reply:
Done